pitivi/render.py

```diff
@@ -369,6 +371,24 @@ def _closeButtonClickedCb(self, unused_button):
     def _playRenderedFileButtonClickedCb(self, unused_button):
         Gio.AppInfo.launch_default_for_uri(self.main_render_dialog.outfile, None)
 
+    def _ShowInFolderButtonClickedCb(self, unused_button):
+        """Pops up the "Show in Folder" dialog box."""
```

Comment:
The folder should be opened with whatever tool the user set to open folders. See the handler for the play button as example.